charts/camunda-platform/values.yaml

```diff
@@ -94,6 +94,10 @@ global:
   zeebeClusterName: "{{ .Release.Name }}-zeebe"
   # ZeebePort defines the port which is used for the Zeebe Gateway. This port accepts the GRPC Client messages and forwards them to the Zeebe Brokers.
   zeebePort: 26500
+  # Multitenancy configuration which is shared between the sub charts
+  multitenancy:
```

Comment:
Are you ok with this global configuration to turn on the multitenancy across components?

Reply:
Yes, all good. That's how it was planned :ok_hand: 
Thank you.